### ISSUE TITLE
Some sections of Bleachit doesn't function

### ISSUE THREAD

**Issue by chdslv** [bleachbit/bleachbit#1196]
Some sections of Bleachit doesn't function. Arch Linux.

```
Exception while running operation 'system': ''utf-8' codec can't encode character '\udcce' in position 63: surrogates not allowed'
Traceback (most recent call last):
  File "/usr/share/bleachbit/Worker.py", line 346, in run_operations
    for dummy in self.clean_operation(operation):
  File "/usr/share/bleachbit/Worker.py", line 160, in clean_operation
    for ret in self.execute(cmd, '%s.%s' % (operation, option_id)):
  File "/usr/share/bleachbit/Worker.py", line 132, in execute
    self.ui.append_text(line)
  File "/usr/share/bleachbit/GUI.py", line 607, in append_text
    self.textbuffer.insert(__iter, text)
  File "/usr/lib/python3.9/site-packages/gi/overrides/Gtk.py", line 764, in insert
    Gtk.TextBuffer.insert(self, iter, text, length)
UnicodeEncodeError: 'utf-8' codec can't encode character '\udcce' in position 63: surrogates not allowed

```

**Comment by abitrolly** [comment]
@chdslv could you be more specific about the version used and clean operation that fails?

**Comment by chdslv** [comment]
> @chdslv could you be more specific about the version used and clean operation that fails?

Bleachbit 4.2.0
![bleachbit](https://user-images.githubusercontent.com/28350701/123057556-4527df80-d408-11eb-93b4-b34bd80190a3.png)

**Comment by abitrolly** [comment]
That's the same Traceback as in https://github.com/bleachbit/bleachbit/issues/1097#issuecomment-766459997 which is supposed to be fixed by https://github.com/bleachbit/bleachbit/pull/951 but appears it is not.

**Comment by chdslv** [comment]
> That's the same Traceback as in [#1097 (comment)](https://github.com/bleachbit/bleachbit/issues/1097#issuecomment-766459997) which is supposed to be fixed by #951 but appears it is not.

Well, I purged Bleachbit and reinstalled, but the problem stays.

**Comment by abitrolly** [comment]
I don't think it is fixed them, but still could you check that it is same with latest version from checkout?
```shell
git clone https://github.com/bleachbit/bleachbit.git
cd bleachbit
./bleachbit.py
```

**Comment by az0** [comment]
I am closing because no reply since June 2021.

[BleachBit 4.4.2 ](https://www.bleachbit.org/news/bleachbit-442)was released November 2021. Please test with this or newer version

**Comment by az0** [comment]
This looks like same issue, so this was fixed in 6.0.0

https://github.com/bleachbit/bleachbit/issues/1709
ac6e8ec08e2f6b4fa6751f59e5b47ddc88083c80
